packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -0,0 +1,144 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { SelectionModeType, SendTokensOptions } from '../../../src/category/account'
+
+describe('SendTokenToAddress', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let addA: string
```

Comment:
you might need 3 addresses to support your results of pie, crumb and forward

Reply:
Yes, i have test two out of the three selection modes. However forward is, in my opinion, not testable because the results are different each time you run the test. Let me know your thoughts.

Reply:
see https://github.com/DeFiCh/jellyfish/pull/323#discussion_r649357820